assets/app/scripts/directives/resources.js

```diff
@@ -45,9 +53,27 @@ angular.module('openshiftConsole')
     return {
       restrict: 'E',
       scope: {
-        pods: '='
+        pods: '=',
+        projectName: '@?' //TODO optional for now
       },
-      templateUrl: 'views/_pods.html'
+      templateUrl: 'views/_pods.html',
+      controller: function($scope) {
+        $scope.phases = [
+          "Failed",
+          "Pending",
+          "Running",
+          "Succeeded"
```

Comment:
@spadgett any opinion on which style to give "Unknown", I'm leaning towards the Failed style or maybe the warning style, although I hesitate to use the warning style since it implies that there is some other thing going on besides the pods phase.  I feel like they should be called out more than the Pending style.